### DISCUSSION TITLE
Integration with Playwright / How to change handlers in test

### DISCUSSION THREAD

**Discussion by florianmatz** [mswjs/msw#1122] [Q&A] [answered]
I'm having trouble to change my handlers in a playwright test like described in the example [for cypress](https://mswjs.io/docs/api/setup-worker/use), because I have no direct access to the window object, or at least I haven't figured out, how to do it yet.

Playwright has some methods like `evaluateHandle` to get kinda access to browser objects, but I haven't managed to get the real window object to access the previously stored msw-stuff.

I know, it's more a Playwright question, but maybe there is somebody who has had similiar problems and maybe found a solution? Any thoughts @kettanaito ?

Would be highly appreciated!

**Comment by d-ivashchuk** [comment, marked as answer]
https://github.com/mswjs/msw/discussions/1322 might be interesting for you @florianmatz
